package.json

```diff
@@ -39,7 +39,7 @@
     "@kubernetes/client-node": "^0.15.1",
     "@snyk/dep-graph": "^1.29.0",
     "async": "^3.2.1",
-    "aws-sdk": "^2.995.0",
+    "aws-sdk": "^2.999.0",
```

Comment:
The lockfile notes a pretty different version! `"version": "2.1019.0",`

Reply:
Yes, I notice that too. I just ran `npm ci`, than `npm install`. I wonder if I should try installing precisely `2.999.0`. I will give that a go. I wondered if this would fix the write EPIPE problem. As I saw in Sentry, seemed like there were loads of write EPIPE error in Snyk https://sentry.io/organizations/snyk/issues/2712543522/?project=64732&referrer=slack